src/diagnostics/VideoTest.js

```diff
@@ -25,6 +25,7 @@ export default class VideoTest extends Test {
 
     this.localMedia.on('localStream', (stream) => {
       if (stream.getVideoTracks().length) {
+        this.streamResolved('we have streams!');
         var videoTrack = stream.getVideoTracks()[0];
         if (videoTrack) {
           this.logger.log('Video stream passed');
```

Comment:
What if we just called `this.localMedia.stop()` here? That way we don't have to store another reference to a deferred promise, and it will still be stopped after passing even if `destroy` isn't called?

Reply:
That is a good idea. I will switch it out. I wasn't crazy about having the deferred promise either.